osu.Game/Overlays/Chat/ChannelControl/ControlItemAvatar.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Online.Chat;
+using osu.Game.Users.Drawables;
+using osuTK;
+
+namespace osu.Game.Overlays.Chat.ChannelControl
+{
+    public class ControlItemAvatar : CircularContainer
+    {
+        private DrawableAvatar? avatar;
+        private readonly Channel channel;
+
+        public ControlItemAvatar(Channel channel)
+        {
+            this.channel = channel;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Size = new Vector2(20);
+            Margin = new MarginPadding { Right = 5 };
+            Masking = true;
+
+            Children = new Drawable[]
+            {
+                new SpriteIcon
+                {
+                    Icon = FontAwesome.Solid.At,
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Colour = Colour4.Black,
+                    RelativeSizeAxes = Axes.Both,
+                    Alpha = 0.2f,
+                },
```

Comment:
This placeholder "at" icon is a really weird detail to add given that if I hadn't read source I wouldn't have even noticed it was there. It may be a good idea to have it but it really ought to be more visible than that imo.

Reply:
This is following the design of the existing placeholder that user avatars have in the current channel tab control.
It looked good enough to keep but I can remove it if it seems superfluous.

Reply:
I'm not saying that it needs to go entirely, just that a 0.2 opacity black "at" sign on top of an already-dark background is doing way less here than what it was doing on the old chat design. Context is important.

Reply:
Adjusted to be faded from white and will hide on avatar load.